### PR TITLE
Add specs for feature feedback - Closes #1161

### LINKED ISSUE #1161: Create feature feedback implementation specification
### Expected behaviour
- specification in PR

### PR DESCRIPTION
**Pull request by faival**
### What was the problem?
missing specs
### How did I fix it?
adding specs
### How to test it?
see PR changes
### Review checklist
- The PR solves #1161 
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

File: docs/1161_SPECS.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+...
+on feedback closed
+if user choosenToBeHIdden 
+	componentObj.choosenToBeHIdden = true
+
+```
+
+#### When to show feature feedback
+
+* on component focus, we set a flag, the flag is updated on every focus event on component. When component is blur, we show feedback if interval is greater than a constant.
```
Comment (michaeltomasik) [inline docs/1161_SPECS.md]: focus event on `the` component

File: docs/1161_SPECS.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      if componentObj.lastGivenOn > todayDate + settings.feedback.showInterval
+      componentObj.lastGivenOn = todayDate;
+      render feature feedback;
+      return;
+
+render component;
+...
+on feedback closed
+if user choosenToBeHIdden 
+	componentObj.choosenToBeHIdden = true
```
Comment (slaweet) [inline docs/1161_SPECS.md]: Little bit of nit-picking: inconsistent capitalization of `choosenToBeHIdden` should be `choosenToBeHidden` here and in few other places.

Comment (faival) [reply docs/1161_SPECS.md]: updated specs

File: docs/1161_SPECS.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+	components: {
+		[componentID]: { 
+			showInterval: [Date/Millisecconds],
+			questions: [Question, ...],
+			lastGivenOn: [Date/Millisecconds],
+		}
+	}
+}
+```
+- **showInterval** = `-1` indicates to show feedback in &#8734; periods, (never repeat)
```
Comment (slaweet) [inline docs/1161_SPECS.md]: I think `undefined` will be more straightforward to use/understand than `-1`

**Pull request merged**